### ISSUE TITLE
Notion icons not displaying properly

### ISSUE THREAD

**Issue by hersheysmcflurry** [notion-enhancer/notion-repackaged#106]
**What is happening?**
Notion's new icons are not displayed properly. Changing colours back and forth for the icons will result in the icons being displayed as blank.

For example,
1. Selecting an icon in grey colour.
2. Changing the same icon to yellow colour.
3. Changing the same icon back to grey colour.

**What should be happening?**
Notion's icons displayed as they are supposed to.

**Screenshots (if applicable):**
First screenshot shows how it looks like on Notion Enhanced (the bug).
<img width="222" alt="Screenshot_2022-08-14_at_12 56 27_PM" src="https://user-images.githubusercontent.com/111222433/184530003-a99b4fd6-cad0-4ced-82bd-344e2b4e0b96.png">
Second screenshot shows how it looks like on Notion iOS (how it's supposed to be).
<img width="222" alt="IMG_1269" src="https://user-images.githubusercontent.com/111222433/184530013-11d57d21-dea6-4c7c-bc66-b4df08934ac6.png">

**Operating system/browser name & version:**
macOS 12.5

**notion-enhancer and/or notion-repackaged version:**
Notion Enhanced 2.0.18-1

**notion-enhancer configuration (enabled/disabled mods and their settings):**
none.

**Comment by developomp** [comment]
Same problem here on Arch Linux.
- Occurs in both [notion-app](https://aur.archlinux.org/packages/notion-app) and [notion-app-enhanced](https://aur.archlinux.org/packages/notion-app-enhanced) package

- DOM when icon is rendered properly
![image](https://user-images.githubusercontent.com/40858122/205007517-bc89377b-7177-472c-8994-528a4a876b2c.png)

- DOM when icon is not rendered properly (just an empty image)
![image](https://user-images.githubusercontent.com/40858122/205007552-8947900f-6c24-4913-9377-9398bce4fee2.png)

- Browser sources only show icons that are rendered properly
![image](https://user-images.githubusercontent.com/40858122/205007593-14d7b6f1-72c5-446f-9c5f-833f933ebe94.png)

**Comment by dragonwocky** [comment]
dup of https://github.com/notion-enhancer/notion-enhancer/issues/770

**Comment by Noob-linux-user** [comment]
can we fix it?

**Comment by dragonwocky** [comment]
@Noob-linux-user I'm working on it.

**Comment by Noob-linux-user** [comment]
ok thank you

**Comment by insignificance** [comment]
same here . macOS 13.1 、Notion enchanted 2.0.18、 Notion 2.1.11

**Comment by lsdrfrx** [comment]
Is there any progress?

**Comment by Bauchet-Tristan** [comment]
Up there because the problem still exist : 
![Capture d’écran 2023-07-18 235659](https://github.com/notion-enhancer/notion-repackaged/assets/74654917/0b3c6168-dd09-4b06-8b85-1c0955c05469)

**Comment by Jorge-lopz** [comment]
Has there been any proggress?

**Comment by dragonwocky** [comment]
@Paco-Rubio yes. You can see progress by checking the commit logs over at https://github.com/notion-enhancer/notion-enhancer/commits/dev/: the version of the notion-enhancer that is in-development has this particular bug fixed in it already, but doesn't have most of the notion-enhancer's themes/extensions updated to be compatible with the new version yet, so it's not ready for release/use until I can get that done.

**Comment by DoctorFrank28** [comment]
notion icons doenst show

**Comment by iliymans** [comment]
> @Paco-Rubio yes. You can see progress by checking the commit logs over at https://github.com/notion-enhancer/notion-enhancer/commits/dev/: the version of the notion-enhancer that is in-development has this particular bug fixed in it already, but doesn't have most of the notion-enhancer's themes/extensions updated to be compatible with the new version yet, so it's not ready for release/use until I can get that done.

so what happened after 3 years?
